lib/data/gateways/music_player/just_audio_player.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:just_audio/just_audio.dart';
+
+import 'music_player_gateway.dart';
+
+class JustAudioPlayer implements MusicPlayerGateway {
+  JustAudioPlayer({
+    required this.justAudio,
+  });
+
+  final AudioPlayer justAudio;
+
+  @override
+  Future<Duration?> loadAudio({required String path}) async {
+    return await justAudio.setAsset(path);
+  }
+
+  @override
+  Future<void> loadPlaylist({
+    required List<String> audioPaths,
+    int initialIndex = 0,
+  }) async {
+    final audioSource = audioPaths
+        .map((assetPath) => AudioSource.uri(Uri.parse('asset:///$assetPath')))
```

Comment:
This URI that i'm using is the same from where the JustAudio look for local uri media